docs/develop-resources/deep-dives/2-define-apis.md

```diff
@@ -37,11 +36,6 @@ go run main.go generate-types \
 
 The proto name of the GCP service, you can find them in [https://github.com/googleapis/googleapis.git](https://github.com/googleapis/googleapis.git). For example, the SQL service is [https://github.com/googleapis/googleapis/tree/master/google/cloud/sql/v1beta4](https://github.com/googleapis/googleapis/tree/master/google/cloud/sql/v1beta4). The `–service` should be `google.cloud.sql.v1beta4`
 
-
-* `--proto-source-path`
```

Comment:
Is this flag removed? Maybe mark it as a optional? But non blocker

Reply:
It is not removed; it's optional to developers because the tool provides the correct default value for it.